src/ducks/triggers/index.js

```diff
@@ -7,7 +7,7 @@ const triggersCollectionKey = 'triggers'
 // CRUD action creators
 
 export const fetchTriggers = () =>
-  cozyClientFetchTriggers(triggersCollectionKey, 'konnector')
+  cozyClientFetchTriggers(triggersCollectionKey)
```

Comment:
I think this commit should not exist anymore. You're now using an optimized way to fetch the triggers, so let's squash it or remove it.